bradybridges/crypto-api
# Seeding

#### What's this PR do?

This PR adds the basic server.js file with endpoints to complete later. Adds initial schema for database. Adds a seed file to populate the database with coindata, portfolio data, and a join table.

#### Where should the reviewer start?

You will need to create a coindata database. Start at the migration and seed file, then look into the basic server.js file.

#### How should this be manually tested?

After creating the coindata db, run the migration file and seed file. Make sure data is as expected with SQL queries.

#### Any background context you want to provide?

Am going the refactor portfolio at a later point, I noticed id and coinId are duplicates. Currently only data for bitcoin is in coindata but will add litecoin and ethereum data later.
